.github/workflows/self-nightly-scheduled.yml

```diff
@@ -212,7 +212,7 @@ jobs:
           python3 -m pip uninstall -y deepspeed
           rm -rf DeepSpeed
           git clone https://github.com/microsoft/DeepSpeed && cd DeepSpeed && rm -rf build
-          DS_BUILD_CPU_ADAM=1 DS_BUILD_FUSED_ADAM=1 DS_BUILD_UTILS=1 python3 -m pip install . --global-option="build_ext" --global-option="-j8" --no-cache -v --disable-pip-version-check
```

Comment:
deepspeed build option `DS_BUILD_UTILS` is deprecated since `v0.10.0` (since https://github.com/microsoft/DeepSpeed/pull/3728)